scripts/clustering.py

```diff
@@ -103,6 +121,7 @@ def main(fmat, kmat, ftags, prefix, fcolor, dimension, pc1, pc2, algorithm, adte
     labels_db = do_clustering.get_cluster_labels()
     n_clusters = do_clustering.get_n_cluster()
 
+
```

Comment:
What changes need to be made to what is saved? Do we just want the cluster labels saved or do you want to also save the projection coordinates?